pymarsys/contact.py

```diff
@@ -32,3 +32,290 @@ def create(self, contact, key_id=None, source_id=None):
             self.endpoint,
             payload=payload
         )
+
+    def create_many(self, contacts, key_id=None):
+        """
+        Create many contacts from a list of dictionaries.
+        :param contacts:  a list of key-value pairs which uniquely identify
+        the contact fields which will be created for the contact (e.g. a key
+        can be the email field ID (3), and its value is the email address of
+        the specific contact).
+        :param key_id: Key which identifies the contacts.
+        :return: Dictionary with a list of the ids of the created contacts.
+        """
+        payload = {
+            'contacts': contacts,
+        }
+        if key_id:
+            payload['key_id'] = key_id
```

Comment:
Yes, only one key_id for all the contacts that will be created. An example from the Emarsys doc:
```
{
  "key_id": "3",
  "contacts":
  [
    {
      "3": "erik.selvig@example.com",
      "2": "Selvig",
      "source_id": "1234"
    },
    {
      "3": "ian.boothby@example.com",
      "2": "Boothby"
    },
    {
      "3": "james.rhodes@example.com",
      "2": "Rhodes",
      "source_id": "5678"
    },
    {
      "3": "pepper.potts@example.com",
      "2": "Potts"
    }
  ]
}
```